presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/BaseRuleTest.java

```diff
@@ -21,11 +22,17 @@
 public abstract class BaseRuleTest
 {
     private RuleTester tester;
+    private Plugin[] plugins;
+
+    public BaseRuleTest(Plugin...plugins)
+    {
+        this.plugins = plugins;
+    }
 
     @BeforeClass
     public final void setUp()
     {
-        tester = new RuleTester();
+        tester = new RuleTester(plugins);
```

Comment:
`RuleTester` is unlikely to benefit from varargs, so  why not `ImmutableList.copyOf(plugins)` here?